app/src/main/java/com/example/cr554/newsapp/GuardianAPI/GuardianService.java

```diff
@@ -0,0 +1,13 @@
+package com.example.cr554.newsapp.GuardianAPI;
+
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.Volley;
+
+/**
+ * Created by cr554 on 2/20/2017.
+ */
+
+public class GuardianService {
+    public static final String API_BASE_URL = "http://content.guardianapis.com/";
+    RequestQueue queue = Volley.newRequestQueue(this);
```

Comment:
RequestQueues should not be created on a per-service / per-class-instance basis. They should, usually, be singletons. Please see this for much more detail (and the reasoning behind why that is the case): https://developer.android.com/training/volley/requestqueue.html#singleton

Here you have an example of such a singleton implemented by me for the tutorial :)

https://github.com/YaleSTC/android_testing_training/blob/master/app/src/main/java/com/example/samplegallery/Utilities/VolleyRequestQueue.java